apps/dashboard/src/@/components/theme-provider.tsx

```diff
@@ -1,9 +1,18 @@
 "use client";
```

Comment:
Just gonna put some things here before to explain why I've replaced a component with hook

* next-themes package only has global context - creating a new Provider does not work
* forcedTheme prop on ThemeProvider does not work with app router